agent/stats/engine_test.go

```diff
@@ -298,7 +302,7 @@ func TestGetTaskHealthMetrics(t *testing.T) {
 	engine.containerInstanceArn = "container_instance"
 
 	containerToStats := make(map[string]*StatsContainer)
-	containerToStats[containerID] = newStatsContainer(containerID, nil, resolver)
+	containerToStats[containerID], _ = newStatsContainer(containerID, nil, resolver)
```

Comment:
should you check assert.NoError here instead of _ ?

Reply:
sure, will add.